src/main/java/ua/danit/final_project/dto/ShiftCommentDto.java

```diff
@@ -27,6 +27,8 @@
 
   private String authorPosition;
 
+  private Long authorId;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "authorId" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=110&issues=AWXJt3cMlgbYnohY1abM&open=AWXJt3cMlgbYnohY1abM)

[ISSUE_KEY]: AWXJt3cMlgbYnohY1abM